.sync/azure_pipelines/RustSetupSteps.yml

```diff
@@ -17,6 +17,30 @@
 
 steps:
 
+- name: Get Host Rust Target Triple
+  id: get_rust_target_triple
+  shell: python
+  run: |
+    import os
+    import platform
+
+    system = platform.system()
+    arch = platform.machine()
+
+    rust_targets = {
+        ('Windows', 'x86_64'): 'x86_64-pc-windows-msvc',
+        ('Windows', 'i386'): 'i686-pc-windows-msvc',

```

Comment:
we will have aarch64 msvc as well?

Reply:
It's possible. I was looking at [tier 1 targets](https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-1-with-host-tools) which doesn't have `aarch64-pc-windows-msvc` (yet).

It is a tier target though so I suppose I could add it now to have it covered. I'll make that update.

Reply:
Updated